### ISSUE TITLE
Allow multiple classes for `no_toc_section_class`

### ISSUE THREAD

**Issue by XhmikosR** [toshimaru/jekyll-toc#70]
For example

```markdown
#### Dealing with specificity

Sometimes contextual classes cannot be applied due to the specificity of another selector. In some cases, a sufficient workaround is to wrap your element's content in a `<div>` with the class.
```

results in this being included in ToC

https://github.com/twbs/bootstrap/blob/9788baedeadddba9d0bcd4dbaea43d65ece9bcbd/site/_includes/callout.html#L1-L9

Demo: https://twbs-bootstrap4.netlify.com/docs/4.1/utilities/colors/

If I change the header to an h5, since we have max h4, then the header is not included in ToC.

/CC @toshimaru

**Comment by XhmikosR** [comment]
Friendly ping @toshimaru :)

I could replace those headers with a header > 5 but ideally this should be fixed.

**Comment by toshimaru** [comment]
@XhmikosR Hi, I checked your site and I found you configured custom class for `no_toc_section`.

https://github.com/twbs/bootstrap/blob/9788baedeadddba9d0bcd4dbaea43d65ece9bcbd/_config.yml#L65

You have two options.

1. Use `bd-example` class instead of `no_toc_section`.
2. Remove your `no_toc_section_class` configuration, and use `no_toc_section`.

I think it's not a bug, so I close the issue, but if any further issue, please let me know.

**Comment by XhmikosR** [comment]
@toshimaru How come you think it's not a bug? I already add the `no_toc_section` in the callout, so I expect any header there to not show up in ToC.

**Comment by toshimaru** [comment]
> * Use `bd-example` class instead of `no_toc_section`.
> * Remove your `no_toc_section_class` configuration, and use `no_toc_section`.

I'd say I already gave options to fix a issue. I'm a not English speaker so if you can't understand the options I gave, could you explain why it doesn't make sense. I could explain the detail more. 👍

**Comment by XhmikosR** [comment]
1. I can't add `bd-example`, that's for other uses
2. Since I can't specify multiple classes there, I'm stuck.

**Comment by toshimaru** [comment]
Hmm, I originally don't want to allow `no_toc_section_class` to have multiple values because it might cause the performance decrement, but I've understood your use case.

I change this issue title and re-open it.

**Comment by XhmikosR** [comment]
I'm not very familiar with Ruby, but it shouldn't have a noticeable performance penalty in theory.

Let me know if you make a beta with this feature.

Thanks!

**Comment by toshimaru** [comment]
@XhmikosR Allowing multiple `no_toc_section` classes has been released at [v0.9.1](https://github.com/toshimaru/jekyll-toc/releases/tag/v0.9.1).

ref. https://github.com/toshimaru/jekyll-toc#skip-toc-section
